local.py

```diff
@@ -2,28 +2,55 @@
 
 import os
 from app import embedder
+from app import classifier
+from app import model_config as mc
+from app import model_fetcher
 from absl import app
 from absl import flags
-import tensorflow_hub as hub
+
+MODEL = "UPR_2percent_ps0"
+INSTANCE = "1573031002"
```

Comment:
why not auto-detect latest?

Reply:
I presume that models may get exported there that aren't deemed "good" so it seemed better to hardcode a known-good model ID.

Reply:
Let's at least add a TODO: mark models as 'released' and automatically pick latest.

Reply:
done